### DISCUSSION TITLE
求黄泉和镜流四命全图

### DISCUSSION THREAD

**Discussion by sanliangdi** [share121/inter-knot#5434] [崩坏：星穹铁道]
如题，求一份全图而不是游戏里那种碎片的，拿来当壁纸用。黄泉和镜流四命图真的好喜欢。谢谢大佬
![19563A4E25122FA5C05C03B9BD028D44_0](https://github.com/user-attachments/assets/2783019b-a2f4-4579-9441-7ad07f5ffb14)

**Comment by Loxia1121** [comment]
有这种东西。。。吗？（）想要有这种效果的话 要么自己画 要么模糊扩图 要不你想想要哪种？后者可以用AI、P图之类的方法搞定（可惜我不会 蹲一下大佬好了（）
